pycaching/cache.py

```diff
@@ -674,14 +684,91 @@ def load_quick(self):
+        'print-page'. Loading as many properties as possible except the
+        following ones, since they are not present on the 'print-page':
+
+          + original_location
+          + state
+          + found
+          + pm_only
+
+        :raise .PMOnlyException: If the PM only warning is shown on the page
+        """
+        if not hasattr(self, "guid"):
+            self._load_guid()
+        res = self.geocaching._request(self._urls["print_page"],
+                                       params={"guid": self.guid})
+        if res.find("p", {"class": "Warning"}) is not None:
+            raise errors.PMOnlyException()
+        content = res.find(id="Content")
+
+        self.name = content.find("h2").get_text()
+
+        self.location = Point.from_string(
+            content.find("p", {"class": "LatLong Meta"}).text)
+
+        type_img = os.path.basename(content.find("h2").img.get("src"))
+        self.type = Type.from_filename(os.path.splitext(type_img)[0])
+
+        size_img = content.find("img", src=re.compile("\/icons\/container\/"))
+        self.size = Size.from_string(size_img.get("alt").split(": ")[1])
+
+        D_and_T_img = content.find(
+            "p", {"class": "Meta DiffTerr"}).find_all("img")
+        self.difficulty, self.terrain = [
+            float(img.get("alt").split()[0]) for img in D_and_T_img
+        ]
+
+        self.author = content.find(
+            "p", text=re.compile("Placed by:")).text.split("\r\n")[2].strip()
+
+        self.hidden = content.find(
+            "p", text=re.compile("Placed Date:")).text.split(": ")[1]
+
+        attr_img = content.find_all("img", src=re.compile("\/attributes\/"))
+        attributes_raw = [
+            _.get("src").split("/")[-1].rsplit("-", 1) for _ in attr_img
+        ]
+        self.attributes = {
+            name: appendix.startswith("yes") for name, appendix
+            in attributes_raw if not appendix.startswith("blank")
+        }
+
+        self.summary = content.find(
+            "h2", text="Short Description").find_next("div").text
+
+        self.description = content.find(
+            "h2", text="Long Description").find_next("div").text
+
+        self.hint = rot13(content.find(id="uxDecryptedHint").text)
+
+        self.favorites = content.find(
+            "strong", text=re.compile("Favorites:")).parent.text.split()[-1]
+
+        self.waypoints = Waypoint.from_html(content, "Waypoints")
+
+    def _load_guid(self):
+        res = self.geocaching._request(self._urls["tiles_server"],
+                                       params={"i": self.wp},
+                                       expect="json")
+
+        if res["status"] == "failed" or len(res["data"]) != 1:
+            msg = res["msg"] if "msg" in res else "Unknown error (probably not existing cache)"
+            raise errors.LoadError("Cache {} cannot be loaded: {}".format(self, msg))
+
+        self.guid = res["data"][0]["g"]
```

Comment:
If you just move this line to `load_quick`, it will have the same effect in the end. 😉